install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -57,6 +57,14 @@ spec:
         (time()-cluster_version_operator_update_retrieval_timestamp_seconds) >= 3600 and ignoring(condition, name, reason) cluster_operator_conditions{name="version", condition="RetrievedUpdates", endpoint="metrics", reason!="NoChannel"}
       labels:
         severity: warning
+    - alert: CurrentVersionNotVerified
+      annotations:
+        summary: The current version of the cluster is not verified.
+        description: The cluster was forcefully upgraded with a provided release image which verification failed, and therefore this cluster may not be trusted or may experience some issues. The verification process does not cover upgradeable checks that depend on the cluster state at the time when the update target was accepted.
```

Comment:
can we explain here how the user would resolve the alert?

Reply:
Yes, that is a good idea.

> The cluster was forcefully upgraded with a provided release image which verification failed, and therefore this cluster may not be trusted or may experience some issues. The verification process does not cover upgradeable checks that depend on the cluster state at the time when the update target was accepted. **To resolve this alert, please upgrade the cluster to a verified version, or silence this alert if you are aware of the potential risks.**

What do you think?

Reply:
I would remove the second part (`, or silence this alert if you are aware of the potential risks.`) since silencing doesn't really resolve the alert (it's still there but hidden).